### PR TITLE
Add inspector integration tests for web

### PR DESCRIPTION
**Pull request by brpins**
## Describe your changes

- Add integration tests for the inspector domain within the web app
- Refactor lpa/validation testing to align with new patterns introduced in this commit

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/views/app/layouts/app-three-column.layout.njk [hunk 1]
```diff
@@ -14,7 +14,7 @@
     {% endif %}
   {% endblock %}
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-one-quarter pins-content--sticky">
+    <div data-test-id="pageLeft" class="govuk-grid-column-one-quarter pins-content--sticky">
```
Comment (radum) [inline apps/web/src/server/views/app/layouts/app-three-column.layout.njk:17]: Should we clear these data test ids when doing a prod release mode? I just don't like them :D

File: apps/web/src/server/lib/request.js [hunk 2]
```diff
@@ -21,9 +21,9 @@ const instance = got.extend({
 	handlers: [
 		createTtlHandler(ttlCache),
 		(options, next) => {
-			if (!options.context.servedFromCache) {
-				console.log(`Sending ${kleur.bgBlue(options.method)} to ${kleur.blue(options.url)}`);
-			}
+			// if (!options.context.servedFromCache) {
```
Comment (radum) [inline apps/web/src/server/lib/request.js:24]: Debugging the issues we had last time was a pain without logging.

Please leave a TODO here to refactor once a more robust logging solution is added.

**Review by radum — APPROVED** [review]

File: apps/web/src/server/views/app/layouts/app-three-column.layout.njk (hunk 1 above)
Comment (brpins) [reply apps/web/src/server/views/app/layouts/app-three-column.layout.njk:17]: What gets deployed to test environment? Is it prod release?

File: apps/web/src/server/lib/request.js (hunk 2 above)
Comment (brpins) [reply apps/web/src/server/lib/request.js:24]: Yeah, I left this commented by mistake, as it's nigh impossible to read test output with this on as the console gets filled with http messages... I think we need a similar kind of debug level thing like for auth

**Pull request merged**